src/Sensors.cpp

```diff
@@ -895,6 +899,24 @@ void Sensors::PMGCJA5Init() {
 void Sensors::dhtInit() {
 }
 
+// Altitude compensation for CO2 sensors without Pressure atm or Altitude compensation
+
+void Sensors::CO2correctionAlt() {
+    //DEBUG("-->[SENSORS] Altitud Compensation for CO2 lectures ON");
+    //Serial.print("-->[SENSORS] CO2 original: ");      // Example test
+    //Serial.print(CO2);                                // Example test
+    //Serial.print(" hpa: ");                           // Example test
```

Comment:
please add these lines with DEBUG macro. With this we have this messages when debugging is enable.

Reply:
done